guides/common/modules/ref_system-requirements-quickstart.adoc

```diff
@@ -26,3 +30,24 @@ ifdef::foreman-el,katello,orcharhino[]
 endif::[]
 * Install {ProductName} on a freshly provisioned system that serves no other function except to run {ProductName}.
 * Ensure the system has at least {project-minimum-memory} of memory.
+
+ifdef::foremanctl[]
+Containerized installation::
+ifdef::foreman-deb[]
+* The following operating systems are supported for deploying {Project}:
+endif::[]
+ifndef::foreman-deb[]
+* The following operating system is supported for deploying {Project}:
+endif::[]
++
+ifndef::foreman-deb[]
+** {EL} 9 (x86_64)
+endif::[]
+ifdef::foreman-deb[]
+** Debian 12 (Bookworm) (amd64)
+** Ubuntu 22.04 (Jammy) (amd64)
+endif::[]
+* Install {ProductName} on a freshly provisioned system that serves no other function except to run {ProductName}.
+* Ensure the system has at least 4 CPU cores and {project-minimum-memory} of memory.
```

Comment:
Should we make this `project-minimum-cpus` or so?
Then we can use it in https://github.com/theforeman/foreman-documentation/blob/ff2788d48c45ce8143f82266c3b4684cbadbc56b/guides/common/modules/ref_system-requirements.adoc?plain=1#L17 too